test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] KIndly make sure the `test.md` file is ignored by git and is not present in your GitHub repository in order to meet all project's requirements :wink: